### PR TITLE
Workaround FUPs.

### PR DESCRIPTION
**Pull request by vext01**
See commit messages.

### CODE REVIEW

File: .buildbot.sh [hunk 1]
```diff
@@ -70,8 +70,12 @@ for mode in "" "--release"; do
     cargo build ${mode} -p ykcapi;
 done
 
-cargo test
-cargo test --release
+# We test lots of times to check for non-deterministic failures.
+for i in `seq 50`; do \
+    cargo test \
+    cargo test --release \
+; done
```
Comment (ltratt) [inline .buildbot.sh:77]: This `; ` isn't necessary if you remove the `\` from the previous line I think?

Comment (vext01) [reply .buildbot.sh]: Yeah, for some reason I wrote a make target in a shell script :P my bad.

Fixed in 0f1c9ba01b51d5c8ba3ec3825c12525be0b528e5

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Comment by vext01** [comment]
That failure is one of the non-determinsitic ones on my TODO. OK to retry for now?

**Comment by ltratt** [comment]
Please force push an update that removes the `.buildbot.sh` change.

**Comment by vext01** [comment]
There we go.

**Pull request merged**